### DISCUSSION TITLE
Looking forward to it!

### DISCUSSION THREAD

**Discussion by scenicfinder** [stadiamaps/ferrostar#35] [Engineering RFDs]
Hi Ian,

Your 'ferrostar' project looks awesome! Your recent blog article was exactly our feelings as my dev has been stuck working with maplibre navigation and mapbox was not going to be viable at all for us financially. 

I know it is very early days, however when do you think it would be safe to implement either the ios / android code into our app? We were working on android but I could always get my dev to switch over to ios development. Alternatively put the project on hold for a bit?

**Comment by ianthetechie** [comment]
Thanks a bunch for your message @scenicfinder! This brings joy to my heart 😍

We're aiming for March/April to have something that I'd call a "production usable MVP." Meaning it probably won't have some of the fancier features (like lane guidance or something), and some of the functionality may be simplistic (ex: off route detection may be mostly distance-based), but it will be 1) stable, and 2) production usable for at least 80% of the common mobile navigation use cases.

At this moment, things are quite chaotic with the codebase, particularly on the Android side. The iOS side is more advanced, but things are still pretty unstable API-wise as we want to move fast and figure out the best ways to express stuff early on before committing to API stability. I'd guess it'll be much less risky (API breakage) to play around in about a month.

**Reply by ianthetechie** [reply]
@scenicfinder we've just launched a GitHub project (kanban board) so you can track the progress to MVP-rediness ;) https://github.com/orgs/stadiamaps/projects/2

**Comment by ianthetechie** [comment]
Developer guide here in case anyone is curious :) It's well past MVP now! https://stadiamaps.github.io/ferrostar/intro.html
